src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -125,16 +126,37 @@
 	public static final RegistryObject<Item> REGIKEY = REGISTER.register("regikey", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.REGI_TIER, 2, -1.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL), new Item.Properties().tab(MSItemGroup.WEAPONS)));
-	public static final RegistryObject<Item> CLOCKKEEPER = REGISTER.register("clockkeeper", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.REGI_TIER, 3, -1.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL), new Item.Properties().defaultDurability(3500).tab(MSItemGroup.WEAPONS).rarity(Rarity.UNCOMMON)));
-	public static final RegistryObject<Item> HOME_BY_MIDNIGHT = REGISTER.register("home_by_midnight", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.REGI_TIER, 3, -1.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).set(PogoEffect.EFFECT_07).add(PogoEffect.EFFECT_07), new Item.Properties().defaultDurability(4000).tab(MSItemGroup.WEAPONS).rarity(Rarity.UNCOMMON)));
+	public static final RegistryObject<Item> CLOCKKEEPER = REGISTER.register("clockkeeper", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.REGI_TIER, 3, -1.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL), new Item.Properties().defaultDurability(3500).tab(MSItemGroup.WEAPONS)));
+	public static final RegistryObject<Item> HOME_BY_MIDNIGHT = REGISTER.register("home_by_midnight", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.REGI_TIER, 4, -1.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).set(PogoEffect.EFFECT_07).add(PogoEffect.EFFECT_07), new Item.Properties().defaultDurability(4000).tab(MSItemGroup.WEAPONS)));
+	public static final RegistryObject<Item> NO_TIME_FOR_FLIES = REGISTER.register("no_time_for_flies", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.CORUNDUM_TIER, 3, -1.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL), new Item.Properties().defaultDurability(2500).tab(MSItemGroup.WEAPONS).rarity(Rarity.UNCOMMON)));
+	public static final RegistryObject<Item> NATURES_HEART = REGISTER.register("natures_heart", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.CACTUS_TIER, 4, -1.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL), new Item.Properties().defaultDurability(2005).tab(MSItemGroup.WEAPONS)));
+	public static final RegistryObject<Item> YALDABAOTHS_KEYTON = REGISTER.register("yaldabaoths_keyton", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.DENIZEN_TIER, 5, -1.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.enemyKnockback(2.0F)).add(OnHitEffect.bossBuster(20, () -> MSEntityTypes.LICH.get())).add(OnHitEffect.playSound(MSSoundEvents.ITEM_BATON_ORCHESTRA)), new Item.Properties().defaultDurability(4000).tab(MSItemGroup.WEAPONS).rarity(Rarity.EPIC)));
```

Comment:
For the playSound OnHitEffect, make sure to use a provider. I.e. 
() -> MSSoundEvents.ITEM_BATON_ORCHESTRA 
instead of just 
MSSoundEvents.ITEM_BATON_ORCHESTRA